<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFlip">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_set">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSet">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter position to change:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter position to change:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_reset">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAll">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QLabel" name="resAll">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>245</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUlong">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>To_Ulong</string>
    </property>
   </widget>
   <widget class="QLabel" name="resTo_Ulong">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUllong">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>To_Ullong</string>
    </property>
   </widget>
   <widget class="QLabel" name="resTo_Ullong">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAny">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Any</string>
    </property>
   </widget>
   <widget class="QLabel" name="resAny">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNone">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="resNone">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>421</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
